packages/flutter/lib/src/services/text_input.dart

```diff
@@ -734,6 +737,15 @@ class TextInputConnection {
     }
     assert(!attached);
   }
+
+  /// User closed the connection.
```

Comment:
This doc is also confusing. This line says the user closed the connection, the next line says its called in responds to a platform notification.

Also, this method should document how this close is different from the close method above and when to call what.

Reply:
done.